connectivity.tf

```diff
@@ -20,3 +20,45 @@ resource "aws_vpc_peering_connection" "bgdc_to_internal_compute" {
     },
   )
 }
+
+data "aws_route_table" "bgdc_private" {
+  subnet_id = tolist(data.aws_subnet_ids.bgdc_private.ids)[0]
+}
+
+resource "aws_route" "bgdc_to_internal_compute" {
+  count                     = length(data.terraform_remote_state.internal_compute.outputs.pdm_subnet.ids)
+  route_table_id            = data.aws_route_table.bgdc_private.id
+  destination_cidr_block    = data.terraform_remote_state.internal_compute.outputs.pdm_subnet.cidr_blocks[count.index]
```

Comment:
Why does BGDC need to route to PDM? The PDM cluster is ephemeral.

Reply:
It's because Hive Metastore RDS sits in PDM subnets